index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>portfolio</title>
+</head>
+<body>
+    <section>
+        <header id="navbar">
+            <a href="#" class="my-logo">My Logo</a>
+            <nav>
+                <img src="image/Union.png" alt="menu">
+            </nav>
+        </header>
+        <main>
+            <h1 class="title">Hi! I am Ezra nice to meet you.</h1>
+            <p class="content">
+                I am a software develper. I can help you build a product,
+                feature or website. Look through some of my work and expereince.
+                If you like what you see and have a project you need coder, don't 
+                husitate to contact me.
+            </p>
+            <h3 class="links">LET'S CONNECT</h3>
+            <div class="contact">
+                <img src="image/social media icons.png" alt="contact">

```

Comment:
Kudos @ezraermy for a job well done 👍.
-  Instead of an image, kindly make use of icons.
-Use `ul` tags to wrap the icons and each icon should be in the `li` tag.

![li](https://user-images.githubusercontent.com/73854999/198389120-da35bdf1-6229-426c-96b9-38ed085d19f5.PNG)